### DISCUSSION TITLE
Using parameters.yml variable inside catalog.yml

### DISCUSSION THREAD

**Discussion by schergui-thefork** [kedro-org/kedro#1694] [Idea]
Hello everyone,

I have a variable that I define inside **parameters.yml** `output_uri: some/u/r/i` and I'd like to use it within **catalog.yml**.

Does anyone know if there is a way to accomplish this ?

Thanks

**Comment by noklam** [comment]
You can achieve this with `TemplatedConfigLoader` where both your `parameters.yml` and `catalog.yml` will be using a variable instead. See https://github.com/kedro-org/kedro/issues/1647

**Reply by schergui-thefork** [reply]
It did work 💪 Thanks

**Reply by noklam** [reply]
awesome!
